docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,113 @@
+# Create Kubernetes cluster using Cleura Cloud Control Panel 
+<!-- Config interpolation for the gui name didn't work for the title here. -->
+
+If you want to create a Kubernetes cluster, you can do it via {{config.extra.gui}} using Gardener.
+This tutorial shows you how to do that, and how to deploy a sample application on such cluster.
+
+## Prerequisites
+
+To access the cluster from your computer, you will need [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine.
+
+## Creating Kubernetes cluster in {{config.extra.gui}}
+
+To get started, navigate to [{{config.extra.gui_domain}}](https://{{config.extra.gui_domain}}) and in the side panel choose Kubernetes
+-> [Managed Kubernetes](https://{{config.extra.gui_domain}}/containers/gardener).
+You will be presented with a Gardener page, in which you can create and manage your clusters. Click `Create Kubernetes cluster`.
+
+![Gardener page in {{config.extra.gui}}](assets/gardener_page.png)
+
+> In Gardener's terminology, kubernetes cluster is referred as a **shoot cluster**. You can see name "Shoot" in various places throughout
+> the panel's UI, so it is good to know what it means. To learn more, check
+> out [Gardener architecture](https://gardener.cloud/docs/gardener/concepts/architecture/).
+
+In the opened form, fill in the name of the cluster and a region to see the rest of the options. Hover over question mark icons at each form
+field to learn more about it. For this tutorial, you can leave everything at default values and click `Create` at the bottom.
```

Comment:
Whoa! No, you can't conflate tutorials and how-tos. What you are writing is a how-to, so please don't call it a tutorial. :) For background, please see <https://diataxis.fr/tutorials-how-to/>.

Reply:
Ahh, this is good to know. I was not aware and sort of "feel" my way into writing how-to, but could be close to actually have features of a tutorial. I know we have a hidden section for tutorials as well though. I think this article could be good to link in contrib page, so that people are aware of the difference.